### PR TITLE
create html folder if doesn't exist

### PR DESCRIPTION
**Pull request by leora-betesh**

### CODE REVIEW

**Comment by szabgab** [comment]
Thanks. Please also add a test covering the new code.

**Comment by leora-betesh** [comment]
How do I do that?  The test case is not having an html folder in the slides repo.

**Comment by szabgab** [comment]
Write a test case that would try to put the html files in a subdirectory of tmpdir (e.g. call it abcd just to make sure we don't overuse the 'html' string) and check if that works properly.

File: test_make_dir.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import os
+
+def make_directory_test(tmpdir):
+    root = str(tmpdir)
+    target_dir = os.path.join(root, 'abcd')
+    generate_html_files(self, target_dir)
+    assert os.path.exists(target_dir) == 1
```
Comment (szabgab) [inline test_make_dir.py]: One minor thing. this should be 
assert os.path.exists(target_dir)
as that method returns True/False

**Comment by szabgab** [comment]
Merged and I made that change myself.

**Comment by szabgab** [comment]
Thanks

**Comment by leora-betesh** [comment]
Got it.  Thanks for the explanation.

**Pull request merged**